Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/DiaryData.kt

```diff
@@ -14,22 +14,26 @@ object DiaryData {
     val LIST_ITEMS = listOf(
         ListItem.Data(
             R.drawable.ic_contact_diary_person_item,
-            "Max Mustermann"
+            "Max Mustermann",
+            ListItem.Type.PERSON
```

Comment:
Caused instrumentation test compile error.

Reply:
Yeah, I think we should inform everyone to check  instrumentation tests on their machines after UI changes.